terraso_backend/apps/graphql/schema/shared_data.py

```diff
@@ -39,6 +39,9 @@ class Meta:
         interfaces = (relay.Node,)
         connection_class = TerrasoConnection
 
+    def resolve_url(self, info):
+        return self.signed_url
```

Comment:
Question: What is the difference between generating the S3 URL here or in the model?

Reply:
I think if we add it here the code while only be called if the filed is in the query

Reply:
We don't want to store the signed URL in the database, since it's temporary.

Reply:
@cacarrara Yes, I see that, my question is (since the signed URL is somwthing that is relevant to the clients) if it worth generating the signed URL at the GraphQL level to avoid generating it even if the url is not needed